app/models/merchant.rb

```diff
@@ -20,10 +20,8 @@ def top_five_customers_by_successful_transaction_count
 
   def self.top_five_by_merchant_revenue
     select("merchants.*, SUM(invoice_items.quantity * invoice_items.unit_price) AS total_rev")
-    .joins(:invoice_items, :transactions)
-    .where(transactions: {result: :success})
-    .group(:id)
-    .order(total_rev: :desc)
+    .joins(:invoice_items, :transactions).where(transactions: {result: :success})
+    .group(:id).order(total_rev: :desc)
     .limit(5)
```

Comment:
Does this work the same as it was written before? Looks clean!

Reply:
i pasted this into rails console and forgot to fix the spacing, will do that next push!